docs/pages/guides/using-eslint.mdx

```diff
@@ -271,4 +298,30 @@ Now you can run `npx expo lint` to lint your code.
 
 </Step>
 
-</TabsGroup>
+## Migration to Flat Config
```

Comment:
Also, after adding this section, do we need the previous migration section: https://docs.expo.dev/guides/using-eslint/#migration-to-eslint-config-expo?

Reply:
The lint command an and `eslint-config-expo` were added in SDK 51, so yeah I think that's probably long enough ago that this can be removed.